### ISSUE TITLE
Base image of Dockerfile python:3.8.1-alpine3.11 contains vulnerable libraries

### ISSUE THREAD

**Issue by BenHimsel** [squidfunk/mkdocs-material#2251]
- [x] I've read the [contribution guidelines][1] and agree with them

__I've found a bug and checked that ...__

- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... the documentation does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem

## Description

Docker image is currently based on python:3.8.1-alpine3.11 which contains outdated packages such as sqlite 3.32.1-r0 which suffers from [CVE-2020-11656](https://nvd.nist.gov/vuln/detail/CVE-2020-11656).

  [1]: https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md

**Comment by BenHimsel** [comment]
[Pull Request 2252](https://github.com/squidfunk/mkdocs-material/pull/2252) submitted with updated python base image.

`mkdocs build` and `mkdocs gh-deploy` validated:

```
/docs # mkdocs build
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /docs/site
INFO    -  Documentation built in 9.34 seconds
/docs # mkdocs gh-deploy
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /docs/site
INFO    -  Documentation built in 6.81 seconds
WARNING -  Version check skipped: No version specified in previous deployment.
INFO    -  Copying '/docs/site' to 'gh-pages' branch and pushing to GitHub.
INFO    -  Your documentation should shortly be available at: https://BenHimsel.github.io/mkdocs-material/
```

**Comment by squidfunk** [comment]
Fixed in #2252

**Comment by squidfunk** [comment]
Released as 6.2.7.
